recipes/dealii/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "dealii" %}
+{% set version = "9.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://dealii.kyomu.43-1.org/downloads/dealii-{{ version }}.tar.gz
+  sha256: df2f0d666f2224be07e3741c0e8e02132fd67ea4579cd16a2429f7416146ee64
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
```

```suggestion
  host:
     - zlib
```

zlib is found from build. (`Found ZLIB: $BUILD_PREFIX/lib/libz.so`) Need to add zlib to host to make deal.ii find it in host instead of build.